systemvm/patches/debian/vpn/etc/ipsec.d/l2tp.conf

```diff
@@ -30,4 +35,5 @@ conn L2TP-PSK
         # ----------------------------------------------------------
         # Change 'ignore' to 'add' to enable this configuration.
         #
+        rightsubnetwithin=10.1.2.0/8
```

Comment:
I thought of keeping /24 but what if the range falls in /20 or /18. I am not sure how much range is used in production in general.